Mathlib/Combinatorics/Enumerative/DyckWord.lean

```diff
@@ -259,11 +259,18 @@ include h in
 lemma firstReturn_pos : 0 < p.firstReturn := by
   by_contra! f
   rw [Nat.le_zero, firstReturn, findIdx_eq] at f
-  · simp only [get_eq_getElem, getElem_range] at f
-    rw [← p.cons_tail_dropLast_concat h] at f
-    simp at f
+  #adaptation_note
+  /--
+  If we don't swap, then the second goal is dropped after completing the first goal.
+  What's going on?
+  -/
```

Comment:
Is this related to the membership mem swap that is now fixed again? So this will be very transient?
If not, shall we add a date string?

Reply:
I'm not sure, unfortunately.